met-web/src/components/engagement/banner/BannerWithImage.tsx

```diff
@@ -2,10 +2,10 @@ import React, { useState } from 'react';
 import { Grid, Box, Typography, Stack } from '@mui/material';
 import { formatDate } from '../../common/dateHelper';
 import BannerWithoutImage from './BannerWithoutImage';
-import { EngagementBannerProps } from '../view/types';
+import { BannerProps } from '../view/types';
```

Comment:
is it assosiated with this email modal change?

Reply:
Yes